pkg/client/fileCheckpointer_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package client
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const checkpointerFilePath = "file_checkpointer_test_file.json"
+
+func after(t *testing.T) {
+	if CheckFileExist(checkpointerFilePath) {
+		err := os.Remove(checkpointerFilePath)
+		require.NoError(t, err)
+	}
+
+}
```

Comment:
Similar to `checkpointer_test`, instead of creating and then deleting a file for every test, it might be easier to create a temp directory up-front, use a new temp file within that directory for each new checkpointer, then delete the whole temp directory at the end.